### ISSUE TITLE
Weird black vertical gradient over image thumbnails in bubble view.

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#21651]
### Steps to reproduce

1. Enable message bubble view
2. Send an image
3. Get very confused that the image thumbnail has a large transparent-to-black vertical gradient overlaid on it - wrecking the thumbnail, given it gets in the way of showing what the image actually should look like.  For instance, when reviewing artwork or photos, the thumbnail now lies - users will see big dark shadows and not know whether they're part of the original design or not (unless they happen to click and zoom).

For instance, this screenshot:

<img width="246" alt="Screenshot 2022-03-31 at 17 30 47" src="https://user-images.githubusercontent.com/1294269/161395619-ce4c174d-809c-467c-90eb-a12e53992327.png">

...ends up looking like...

<img width="259" alt="Screenshot 2022-04-02 at 18 59 30" src="https://user-images.githubusercontent.com/1294269/161395623-8ce66c8a-8db8-4194-9747-937ddc8c2e8c.png">

#### What did you expect?

Thumbnails to show a faithful representation of the full resolution image

#### What happened instead?

Thumbnail shows big black gradient, presumably to ensure the timestamp is visible, but completely at the expense of the fidelity of the thumbnail, severely impacting its usability.

### Operating system

macOS

### Application version

Nightly

### How did you install the app?

Nightly

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by ShadowJonathan** [comment]
FWIW applications like telegram use small pills when mousing over the image, or at least start their gradient (iirc) only in the last 10% of that image; 
![image](https://user-images.githubusercontent.com/22740616/161396015-969e9e87-8828-4e26-966c-00ab6c45678f.png)

**Comment by robintown** [comment]
See https://github.com/vector-im/element-web/issues/20402 for context of why the gradient exists (yes, it is for timestamp legibility)

**Comment by robintown** [comment]
Yeah, and actually, I still find the gradient a bit insufficient for legibility against certain images, so a plinth would be welcome.

**Comment by ara4n** [comment]
fixed in favour of a plinth, thank heavens

**Comment by t3chguy** [comment]
Hmm it seems like it is now left with nothing 

![image](https://user-images.githubusercontent.com/2403652/169695448-f47301ba-e2e4-4225-88e9-ff3c0f5bcf1d.png)

Which proves rather difficult to read

Should probably get the same banner/pill thing that the file name & size get

**Comment by t3chguy** [comment]
The code still seems to contain the linear-gradient, it simply regressed

![image](https://user-images.githubusercontent.com/2403652/169695640-ad65a127-402e-4d3a-8de2-e6d8e403e2ed.png)

**Comment by robintown** [comment]
The gradient is still very much there, it's just always been subtle on images with a non-white background

![Screenshot 2022-05-22 at 09-08-39 Element 4 Test room](https://user-images.githubusercontent.com/48614497/169696686-d78c475f-3014-4803-b354-c99364ef4e45.png)

**Comment by ara4n** [comment]
oh :( plz can kill with 🔥

**Comment by t3chguy** [comment]
In its current state it is broken, only works on transparent images so the layering order is wrong.

**Comment by daniellekirkwood** [comment]
Might be related: https://github.com/vector-im/element-web/issues/23409
